### DISCUSSION TITLE
ALVR v15.2.1

### DISCUSSION THREAD

**Discussion by zmerp** [alvr-org/ALVR#628] [General]
* Support high frequency hand tracking (by @nachog2003)
* Add Qualcomm low latency flag for the decoder (by @Armarr)

<hr /><em>This discussion was created from the release <a href='https://github.com/alvr-org/ALVR/releases/tag/v15.2.1'>ALVR v15.2.1</a>.</em>

**Comment by alvrdoesntfuckingwork** [comment]


**Reply by alvrdoesntfuckingwork** [reply]
agreed

**Reply by alvrdoesntfuckingwork** [reply]
> agreed

forget to switched accounts op?

**Reply by alvrdoesntfuckingwork** [reply]
> > agreed
> 
> forget to switched accounts op?

HAHAHAHA what

![image](https://user-images.githubusercontent.com/83897525/117561933-8767ae00-b060-11eb-8da9-9bb53d529d3c.png)

**Reply by zmerp** [reply]
Please do not spam. Either provide constructive criticism or do not even post.

**Comment by HeadClot** [comment]
Hey - I just did some quick testing and It seems that previous version of ALVR Works (aka v15.2.0) but this version does not work. It crashes. I am not sure what is going on with this version. Either way do not use this version.
